src/admin.njk

```diff
@@ -0,0 +1,25 @@
+---
+permalink: '/admin/index.html'
+---
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Content Manager</title>
+  <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+</head>
+<body>
+  <!-- Include the script that builds the page and powers Netlify CMS -->
+  <script src="https://unpkg.com/netlify-cms@2.10.103/dist/netlify-cms.js"></script>
+	{% if env.context === "dev" %}
```

Comment:
Please sync up the use of tabs and spaces for indentation. Line 15-21 use tabs for indentation while other lines use 2 spaces. Please use tabs across.

This comment applies to other files too. Thanks.